style/layout/_main.scss

```diff
@@ -0,0 +1,197 @@
+        }
+
+        &--2 {
+            background: url('../../img/leaf-home.svg');
+            background-size: cover;
+            width: 52rem;
+            height: 20rem;
+            margin: 2rem 2rem 2rem 2rem;
+        }
+
+        &--3 {
+            background: url('../../img/meal.svg');
+            background-size: cover;
+            width: 52rem;
+            height: 20rem;
+            margin: 2rem 2rem 2rem 2rem;
+        }
+    }
+
+    &__textbox {
+        display: block;
+        margin: 2rem 0;
+    }
+
+    &__subheading {
+        color: $color-primary;
+        font-family: $primary-font;
+        font-size: $font-size-small;
+        text-transform: uppercase;
+        font-weight: 600;
+        margin: 2rem 0;
+    }
+
+    &__heading {
+        font-size: 3rem;
+        margin: 0;
+    }
+
+    &__paragraph {
+        margin: 2rem 0;
+        line-height: 2;
+    }
+
+    &__morebox {
+        position: relative;
+    }
+
+    &__more {
+        font-family: $primary-font;
+        color: $color-black;
+        font-size: $font-size-small;
+        font-weight: 600;
+        text-decoration: none;
+        text-transform: uppercase;
+        margin-left: 4rem;
+
+        &::before {
+            content: "+";
+            font-family: $primary-font;
+            font-size: $font-size-big;
+            color: $color-primary;
+            display: inline-block;
+            transform: rotate(-45deg);
+            position: absolute;
+            bottom: -2rem;
+            left: 0;
+        }
+    }
+}
+
+.mission {
+    margin: 2rem;
+    margin-top: 6.5rem;
+    max-width: 40%;
+    position: relative;
+
+    &__back-subheading {
+        font-family: $primary-font;
+        color: $color-white;
+        font-size: $font-size-xxl;
+        font-weight: 500;
+        text-transform: uppercase;
+        word-break: break-all;
+        line-height: 0.7;
+        max-width: 40rem;
+
+        &::before {
+            content: "01";
+            font-family: $primary-font;
+            font-size: 4.8rem;
+            font-weight: 600;
+            color: $color-primary;
+            display: inline-block;
+            position: absolute;
+            left: -4rem;
+            top: -2rem;
+        }
+    }
+
+    &__back-heading {
+        font-family: $secondary-font;
+        color: $color-primary;
+        font-size: 8rem;
+        position: absolute;
+        transform: rotate(-20deg);
+        top: 7rem;
+        right: 6rem;
+    }
+
+    &__img {
+        width: 50rem;
+        position: absolute;
+        top: 12rem;
+        left: -8rem;
+    }
+
+    &__textbox {
+        margin: 27rem 4rem 2rem 4rem;
+        position: relative;
+    }
+
+    &__heading {
+        font-family: $primary-font;
+        font-size: 5rem;
+        font-weight: 600;
+        margin-bottom: 2rem;
+    }
+
+    &__subheading {
+        font-size: 3rem;
+        max-width: 30rem;
+        margin-bottom: 2rem;
+    }
+
+    &__paragraph {
```

Comment:
Szczerze to ta nazwa "paragraph" też mi się nie podoba, bo znów kolaży się z tagiem "p", lepiej używać nazw klas mówiących co to jest, aniżeli tagi, czy wygląd.

Reply:
Ok, to zamienie to na coś innego tylko nie wiem na co za bardzo...

Reply:
"description" np?